src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -0,0 +1,443 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from datetime import datetime
+import json
+import logging
+import os
+import jwt
+
+import sas
+import sas_testcase
+from util import winnforum_testcase, configurable_testcase, writeConfig, \
+  loadConfig, generateCpiRsaKeys, generateCpiEcKeys, convertRequestToRequestWithCpiSignature
+
+
+class ExclusionZoneTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_EXZ_1_default_config(self, filename):
+    """Generates the WinnForum configuration for EXZ.1."""
+    # Load device info
+    # Load Devices located outside 50 meters of exclusion zones
+    device_N2_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_N2_1['installationParam']['latitude'] = 40.31304
+    device_N2_1['installationParam']['longitude'] =  -96.25122
+    device_N2_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_N2_2['installationParam']['latitude'] = 40.35491
+    device_N2_2['installationParam']['longitude'] =  -96.4215
+    device_N2_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_N2_3['installationParam']['latitude'] = 40.40931
+    device_N2_3['installationParam']['longitude'] =  -96.09191
+
+    # Load Devices located within at least one exclusion zones
+    device_N3_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
```

Comment:
You need to alter the FCC ID and/or serial number if you're going to re-load the same registration request (here and below). Otherwise it will look like we are trying to re-register the same CBSD.

Reply:
Altered the fcc id and serial number for N3,N4